### PR TITLE
Plugin/list issues from last milestone

### PR DESCRIPTION
**Pull request by dzajkowski**
Introduces:
- github effect
- github based issues plugin
- pure implementation of github effect
- simple implementation of issues listing

### CODE REVIEW

File: src/Frost/Effects/Github.hs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+makeSem ''Github
+
+runGithubPure :: [Issue] -> Sem (Github ': r) a -> Sem r a
+runGithubPure issues = interpret $ \case
+  Issues _ -> return issues
+
+runGithubIO :: (Member (Embed IO) r, Member (Error FrostError) r) => Sem (Github ': r) a  -> Sem r a
+runGithubIO = interpret $ \case
+  Issues repo -> do
+    (username, reponame) <- either throw (\b -> return b) (parseRepo repo)
```
Comment (rabbitonweb) [inline src/Frost/Effects/Github.hs]: Change ` (username, reponame) <- either throw (\b -> return b) (parseRepo repo)` to 
` (username, reponame) <- either throw return (parseRepo repo)`

File: src/Frost/Effects/Github.hs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    fromEitherSem $ embed $ issuesForFrost username reponame
+  where
+    issuesForFrost username reponame = do
+      mis <- GHI.issuesForRepo username reponame GHO.optionsAnyMilestone
+      return $ either (\e -> Left $ FrostError $ show e) (\l -> Right $ fmap (\i -> show $ GHDI.issueTitle i) (DV.toList l) ) mis
```
Comment (rabbitonweb) [inline src/Frost/Effects/Github.hs]: `(\e -> Left $ FrostError $ show e)` can be just function composition `Left . FrostError . show`

Comment (rabbitonweb) [reply src/Frost/Effects/Github.hs]: `(\l -> Right $ fmap (\i -> show $ GHDI.issueTitle i) (DV.toList l) ) ` I think [not sure] can be just
`Right . (fmap (show . GHDI.issueTitle ) . DV.toList`

File: src/Frost/Plugins/LatestIssuesPlugin.hs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+import Frost.Plugin
+import Text.Pandoc
+import Polysemy
+import PolysemyContrib
+import Data.Map.Strict
+import Frost.Effects.Github
+import Data.List
+
+latestIssuesPlugin :: (Member Github r) => Plugin r
```
Comment (rabbitonweb) [inline src/Frost/Plugins/LatestIssuesPlugin.hs]: Maybe it's better to inline `bl`

```
latestIssuesPlugin :: (Member Github r) => Plugin r
latestIssuesPlugin = justContentPlugin "issues:latest" do
        is <- issues repo
        return $ (renderBlock is, renderInline is)
  where
  (...)
```

Comment (rabbitonweb) [reply src/Frost/Plugins/LatestIssuesPlugin.hs]: but not sure, I guess it's matter of style

**Comment by lukasz-golebiewski** [comment]
Are you planning to add any tests?

**Pull request merged**